modules/install-sno-about-installing-single-node-openshift.adoc

```diff
@@ -0,0 +1,10 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-preparing-to-install-sno.adoc
+
+[id="install-sno-about-installing-single-node-openshift_{context}"]
+= About installing Single Node OpenShift (SNO)
+
+Single Node OpenShift (SNO) is a specialized installation. The primary use case is for edge computing workloads, including small physical footprints, intermittent connectivity, portable clouds, and 5G radio access networks (RAN) close to a base station. An installation on a single node supports autonomous workloads.
+
+The major trade off with an installation on a single node is the lack of high availability. Currently, Red Hat does not support upgrading {product-title} when installed on a single node. Upgrades and some configuration changes require re-installation.
```

Comment:
I think we do support upgrades.
@romfreiman ?

Reply:
Confirmed by https://bugzilla.redhat.com/show_bug.cgi?id=2017242